lib_static/open_project/configuration/helpers.rb

```diff
@@ -131,13 +131,17 @@ def use_iam_profile?
       end
 
       def fog_credentials
-        (Hash(self["fog"])["credentials"] || {}).map { |key, value| [key.to_sym, value] }.to_h
+        (Hash(self["fog"])["credentials"] || {}).transform_keys(&:to_sym).to_h
```

Comment:
:green_circle: What about

```suggestion
        (Hash(self["fog"])["credentials"] || {}).symbolize_keys
```

Reply:
Yep, even better 👍

Reply:
And it also fixes an issue that I do not quite understand:

```
>> Hash(self["fog"])["credentials"] || {}
=> {"provider" => "AWS"}
>> (Hash(self["fog"])["credentials"] || {}).transform_keys(&:to_sym)
=> {"provider" => "AWS"}
>> {"provider" => "AWS"}.transform_keys(&:to_sym)
=> {provider: "AWS"}
```

I would have expected `(Hash(self["fog"])["credentials"] || {}).transform_keys(&:to_sym)` to return `{provider: "AWS"}`, not `{"provider" => "AWS"}`.

Reply:
Can you do `Hash(self["fog"])["credentials"].class`?

I bet it's not a hash, but something that _mostly_ quacks like a hash...

Reply:
```
[2] pry(main)> h = HashWithIndifferentAccess.new
=> {}
[3] pry(main)> h[:foo] = 42
=> 42
[4] pry(main)> h
=> {"foo" => 42}
[5] pry(main)> h.transform_keys(&:to_sym)
=> {"foo" => 42}
[6] pry(main)> h.to_h
=> {"foo" => 42}
[7] pry(main)> h.symbolize_keys
=> {foo: 42}
[8] pry(main)> h.symbolize_keys.class
=> Hash
```

Reply:
You were faster than me. I was about to post the same kind of things.
Thanks for looking into it 👍